src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -1,12 +1,18 @@
 import { Component, EventEmitter, Input, OnInit, Output, ViewChild } from '@angular/core';
 import { AbstractControl, FormGroup } from '@angular/forms';
-import { FILE_TYPE_SUPPORT_ERROR, SUPPORTED_FILE_TYPES, UploadService } from '../../core/services/upload.service';
+import {
+  FILE_TYPE_SUPPORT_ERROR,
+  SUPPORTED_FILE_TYPES,
+  UploadService,
+  getSizeExceedErrorMsg } from '../../core/services/upload.service';
 import { ErrorHandlingService } from '../../core/services/error-handling.service';
 import { HttpErrorResponse } from '@angular/common/http';
 import { ElectronService } from '../../core/services/electron.service';
 
 const DISPLAYABLE_CONTENT = ['gif', 'jpeg', 'jpg', 'png'];
-const MAX_UPLOAD_SIZE = 10000000; // 10MB
+const BYTES_PER_MB = 100000;
```

Comment:
I think its supposed to be 1000000, its probably causing the error I encountered.

![image](https://user-images.githubusercontent.com/39243082/110920683-a267bb80-8358-11eb-9d7e-c4f68a4c79e7.png)

```suggestion
const BYTES_PER_MB = 1000000;
```